test/startservers.py

```diff
@@ -1,120 +1,388 @@
-        ('boulder-sa-1', 'boulder-sa-2')),
-    Service('akamai-test-srv',
-        6789, None,
-        ('./bin/akamai-test-srv', '--listen', 'localhost:6789', '--secret', 'its-a-secret'),
-        None),
-    Service('akamai-purger',
-        9666, None,
-        ('./bin/boulder', 'akamai-purger', '--config', os.path.join(config_dir, 'akamai-purger.json')),
-        ('akamai-test-srv',)),
-    Service('s3-test-srv',
-        7890, None,
-        ('./bin/s3-test-srv', '--listen', 'localhost:7890'),
-        None),
-    Service('crl-storer',
-        9667, None,
-        ('./bin/boulder', 'crl-storer', '--config', os.path.join(config_dir, 'crl-storer.json')),
-        ('s3-test-srv',)),
-    Service('crl-updater',
-        8021, None,
-        ('./bin/boulder', 'crl-updater', '--config', os.path.join(config_dir, 'crl-updater.json')),
-        ('boulder-ca-a', 'boulder-ca-b', 'boulder-sa-1', 'boulder-sa-2', 'crl-storer')),
-    Service('boulder-ra-1',
-        8002, 'ra1.service.consul:9094',
-        ('./bin/boulder', 'boulder-ra', '--config', os.path.join(config_dir, 'ra.json'), '--addr', 'ra1.service.consul:9094', '--debug-addr', ':8002'),
-        ('boulder-sa-1', 'boulder-sa-2', 'boulder-ca-a', 'boulder-ca-b', 'boulder-va-1', 'boulder-va-2', 'akamai-purger', 'boulder-publisher-1', 'boulder-publisher-2')),
-    Service('boulder-ra-2',
-        8102, 'ra2.service.consul:9094',
-        ('./bin/boulder', 'boulder-ra', '--config', os.path.join(config_dir, 'ra.json'), '--addr', 'ra2.service.consul:9094', '--debug-addr', ':8102'),
-        ('boulder-sa-1', 'boulder-sa-2', 'boulder-ca-a', 'boulder-ca-b', 'boulder-va-1', 'boulder-va-2', 'akamai-purger', 'boulder-publisher-1', 'boulder-publisher-2')),
-    Service('bad-key-revoker',
-        8020, None,
-        ('./bin/boulder', 'bad-key-revoker', '--config', os.path.join(config_dir, 'bad-key-revoker.json')),
-        ('boulder-ra-1', 'boulder-ra-2', 'mail-test-srv')),
-    Service('nonce-service-taro',
-        8111, 'nonce1.service.consul:9101',
-        ('./bin/boulder', 'nonce-service', '--config', os.path.join(config_dir, 'nonce-a.json'), '--addr', '10.77.77.77:9101', '--debug-addr', ':8111',),
-        None),
-    Service('nonce-service-zinc',
-        8112, 'nonce2.service.consul:9101',
-        ('./bin/boulder', 'nonce-service', '--config', os.path.join(config_dir, 'nonce-b.json'), '--addr', '10.88.88.88:9101', '--debug-addr', ':8112',),
-        None),
-    Service('boulder-wfe2',
-        4001, None,
-        ('./bin/boulder', 'boulder-wfe2', '--config', os.path.join(config_dir, 'wfe2.json')),
-        ('boulder-ra-1', 'boulder-ra-2', 'boulder-sa-1', 'boulder-sa-2', 'nonce-service-taro', 'nonce-service-zinc')),
-    Service('log-validator',
-        8016, None,
-        ('./bin/boulder', 'log-validator', '--config', os.path.join(config_dir, 'log-validator.json')),
-        None),
+    Service(
```

Comment:
In many areas I feel like the consistency added by `black` (I assume) is actually improving readability. Here and a couple areas below though, I think it is doing more harm than good.

Reply:
https://github.com/letsencrypt/boulder/compare/main...mcpherrinm:boulder:mattm-python-long?expand=1

I tried again with a longer line length to see if that helps

Reply:
It's definitely better, closing this version.